webapp/src/components/chat/prompt-dialog/PromptDialog.tsx

```diff
@@ -50,18 +53,20 @@ export const PromptDialog: React.FC<IPromptDialogProps> = ({ message }) => {
     } catch (e) {
         prompt = message.prompt ?? '';
     }
-
     let promptDetails;
     if (typeof prompt === 'string') {
         promptDetails = prompt.split('\n').map((paragraph, idx) => <p key={`prompt-details-${idx}`}>{paragraph}</p>);
     } else {
         promptDetails = Object.entries(prompt).map(([key, value]) => {
+            const isStepwiseThoughtProcess = Constants.STEPWISE_RESULT_NOT_FOUND_REGEX.test(value as string);
             return value ? (
                 <div className={classes.prompt} key={`prompt-details-${key}`}>
                     <Body1Strong>{PromptSectionsNameMap[key]}</Body1Strong>
-                    {(value as string).split('\n').map((paragraph, idx) => (
-                        <p key={`prompt-details-${idx}`}>{paragraph}</p>
-                    ))}
+                    {isStepwiseThoughtProcess ? (
+                        <StepwiseThoughtProcess stepwiseResult={value as string} />
```

Comment:
it looks like this can get pretty long. is a scrollbar needed for this and if so, does it show?

Reply:
Yeah it doesn't change the behavior / UX that we have for the prompt dialog today, which is basically it stretches to fit the entire screen and then you can scroll in the dialog
![image](https://github.com/microsoft/chat-copilot/assets/125500434/9039401c-7d9b-4963-8067-23bd43d5d041)